easy-ui-react/src/ToggleCard/ToggleCard.tsx

```diff
@@ -124,8 +122,8 @@ function ToggleCardBody(props: ToggleCardBodyProps) {
   const { children } = props;
 
   return (
-    <Card.Area padding="0.5">
-      <div className={styles.body}>{children}</div>
+    <Card.Area paddingX="0.5" paddingY="1">
```

Comment:
oh wait i think the padding on the `ToggleCardHeader` should be `paddingY="0.5" paddingX="1"`:

<img width="720" alt="image" src="https://github.com/user-attachments/assets/7f277534-f973-4f73-8b13-94b2d8664024">

Reply:
ah! good catch, i assumed the red line at the bottom applied to the top, but yeah, looks like its intended to be `paddingY="0.5"` `paddingX="1"`